ucc/project/client/Client.java

```diff
@@ -0,0 +1,38 @@
+package org.example.client;
+
+import java.io.BufferedReader;
+import java.io.FileReader;
+
+public class Client {
+
+    public static void main(String[] args) {
+       readFile();
+    }
+
+    public static void readFile() {
+        // Will read a client request from a file
+
+        try {
+            // BufferedReader to read text from an input stream
+            BufferedReader bufferedReader = new BufferedReader(new FileReader("src/main/java/org/example/client/file.txt"));
```

Comment:
this is probably fine for now, but we should avoid hard coding values. A better approach is to use unix 
redirection 

[https://www.tutorialspoint.com/unix/unix-io-redirections.htm](url)

 java Client < input.txt > output.txt

This way we avoid any hardcoding and the freedom to pass in any file.

```Java
 try {
        BufferedReader input = new BufferedReader(new InputStreamReader(System.in));
        String line;
        while ((line = input.readLine()) != null) {
            System.out.println("Value, " + line + "line 1");
        }   
    } catch (IOException ioex) {
        ioex.printStackTrace();
    }   

something like the above will work